packages/desktop-client/src/components/modals/ImportTransactionsModal/FieldMappings.jsx

```diff
@@ -6,6 +6,7 @@ import { SectionLabel } from '../../forms';
 
 import { SelectField } from './SelectField';
 import { SubLabel } from './SubLabel';
+import { stripCsvImportTransaction } from './utils';
```

Comment:
nit: this would look cleaner in a new section if lint will allow

```suggestion

import { stripCsvImportTransaction } from './utils';
```

Reply:
Lint doesn't allow it :/

Reply:
No problem :)